pandas/core/sorting.py

```diff
@@ -385,7 +385,7 @@ def ensure_key_mapped(values, key: Optional[Callable], levels=None):
     from pandas.core.indexes.api import Index
 
     if not key:
-        return values.copy()
+        return values
```

Comment:
why is this being changed ?

Reply:
problem with merging branches. sorry for that